hassio/src/supervisor-base-element.ts

```diff
@@ -49,20 +51,71 @@ export class SupervisorBaseElement extends urlSyncMixin(
     Collection<unknown>
   > = {};
 
+  @internalProperty() private _resources?: Record<string, any>;
+
+  @internalProperty() private _language = "en";
+
+  public connectedCallback(): void {
+    super.connectedCallback();
+    this._initializeLocalize();
+  }
+
   public disconnectedCallback() {
     super.disconnectedCallback();
     Object.keys(this._unsubs).forEach((unsub) => {
       this._unsubs[unsub]();
     });
   }
 
+  protected updated(changedProperties: PropertyValues) {
+    super.updated(changedProperties);
+
+    if (changedProperties.has("_language")) {
+      if (changedProperties.get("_language") !== this._language) {
+        this._initializeLocalize();
+      }
+    }
+
+    if (
+      this._language &&
+      this._resources &&
+      (changedProperties.has("_language") ||
+        changedProperties.has("_resources"))
```

Comment:
This means that we will `computeLocalize` twice when the language changes, as the `_initializeLocalize` from line 75 will also create an update when it has loaded the resources?